node/src/components/linear_chain.rs

```diff
@@ -67,13 +67,13 @@ pub enum Event<I> {
         /// The deploys' execution results.
         execution_results: HashMap<DeployHash, ExecutionResult>,
     },
-    /// The result of requesting a block from storage to add a finality signature to it.
-    GetBlockForFinalitySignaturesResult(Box<FinalitySignature>, Option<Box<Block>>),
+    /// The result of requesting finality signatures from storage to add pending signatures.
+    GetStoredFinalitySignaturesResult(Box<FinalitySignature>, Option<Box<BlockSignatures>>),
```

Comment:
I am not sure we want this to be a `Box<FinalitySignature>`.  It might be more appropriate to have:

```rust
GetStoredFinalitySignaturesResult {
  block_hash: BlockHash, 
  era_id: u64,
  block_signatures: Box<BlockSignatures> 
}
```

that might be a lift, however.